fp.js

```diff
@@ -6,9 +6,14 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(person => {
-  peoplesAges.push(person.age)
-})
+
+var newAges = people.map(function(age){
+  return age;
+});
```

Comment:
not quite, very close though. `age` is a property on a `person` ( each object in the array `people`), so we need to modify our function like so:

```js
// es6
let newAges = people.map( person => person.age)

//es5
var newAges = people.map( function (person)  { return person.age })
```